src/components/Button.js

```diff
@@ -13,11 +14,16 @@ const Button = props => {
 
   let expand;
 
-  props.wide ? (expand = { width: '50%' }) : (expand = { width: '25%' });
+  wide ? (expand = { width: '50%' }) : (expand = { width: '25%' });
 
   return (
-    <div className='button' style={{ ...bg, ...expand }}>
-      {props.names}
+    <div
+      onClick={() => clickHandler(names)}
+      value={names}
+      className='button'
+      style={{ ...bg, ...expand }}
```

Comment:
See my comment above for putting all styles into the `Button.scss` file instead of here. We can indeed do styling inline in React. But then again since we are already importing a Button.scss module anyway, we might as well be consistent.